docs/integrations/sources/adroll.md

```diff
@@ -0,0 +1,45 @@
+# AdRoll
+AdRoll is an ecommerce marketing platform that gives growing D2C brands the power to connect with customers wherever they are.
+This connector allows you to extract data from various AdRoll APIs such as Advertisables , Ads , Campaigns , Strategies and many more
+Docs : https://developers.adroll.com/apis
+
+## Configuration
+
+| Input | Type | Description | Default Value |
+|-------|------|-------------|---------------|
+| `Authorization` | `string` | Person Access Token (Authorization).  |  |
+| `client_id` | `string` | Client ID. The key ID of your app |  |
+| `client_secret` | `string` | Client secret. The secret of your app |  |
+| `client_refresh_token` | `string` | Refresh token.  |  |
+| `oauth_access_token` | `string` | Access token. The current access token. This field might be overridden by the connector based on the token refresh endpoint response. |  |
+| `oauth_token_expiry_date` | `string` | Token expiry date. The date the current access token expires in. This field might be overridden by the connector based on the token refresh endpoint response. |  |
+
+## Streams
+| Stream Name | Primary Key | Pagination | Supports Full Sync | Supports Incremental |
+|-------------|-------------|------------|---------------------|----------------------|
+| Segments | segment_id | DefaultPaginator | ✅ |  ❌  |
```

Comment:
The pagination for `segments` is wrong. This endpoint doesn't support pagination. 
If you change the Page Size parameter to 2 you'll see the pagination breaks becasue never ends.

Docs: https://developers.adroll.com/docs/adroll-audience-api/1/routes/segments/get

Reply:
Sorry for forcefully trying to add pagination to this stream . It had worked hence I had pushed. I will remove it !! 🙇

Reply:
I will remove pagiantion from streams using reporting APIs  as well becuase even they dont support it officially

Reply:
DONE